src/AvaloniaEdit/Search/SearchPanel.cs

```diff
@@ -456,8 +451,6 @@ private void SearchLayerKeyDown(object sender, KeyEventArgs e)
                     Close();
                     break;
             }
-
-            e.Handled = true;
```

Comment:
Why this change?

Reply:
In previous version, Search panel doesn't not take any input from keyboard